container/shim/src/bin/shim.js

```diff
@@ -59,7 +60,7 @@ trapServer(server);
 
 async function shutdown() {
   try {
-    await Promise.allSettled([submitRetrievals(), ORCHESTRATOR_REGISTRATION ? deregister() : Promise.resolve()]);
+    await Promise.allSettled([startLogIngestor(), ORCHESTRATOR_REGISTRATION ? deregister() : Promise.resolve()]);
```

Comment:
The timeout prevents dual interval, but not dual execution, what happens if it's already running when this is called?

Reply:
addressed in https://github.com/filecoin-saturn/L1-node/pull/187/commits/78060a780cee5e8d889400ffd7237779588dfbb4